ci/kokoro/ubuntu/build.sh

```diff
@@ -38,6 +38,11 @@ build:results-local --auth_scope=https://www.googleapis.com/auth/cloud-source-to
 build:results-local --experimental_remote_spawn_cache
 _EOF_
 
+# Kokoro does guarantee that g++-4.9 will be installed, but the default compiler
```

Comment:
But if it's not installed, how can you be sure you can just point to it, and that they will be at those paths?

Or do you mean it's not guaranteed to be pre-installed?

Reply:
OK, thanks for the clarification.